renovate.json

```diff
@@ -26,7 +26,6 @@
       "description": "Group minor and patch updates into a single PR",
       "groupName": "dependencies",
       "managers": [
-        "terraform",
```

Comment:
FYI, we might be able to kill the `managers` key entirely - it's now called [`"enabledManagers"`](https://docs.renovatebot.com/configuration-options/#enabledmanagers) because if you set a value for it, you're disabling all the managers not in the list.

Reply:
Good catch, removed in latest commit.